torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -578,6 +578,8 @@ def make(value, source=None, **kwargs):
             return OutDtypeHigherOrderVariable(value, source, **kwargs)
         elif value.__name__ == "wrap":
             return WrapHigherOrderVariable(value, source, **kwargs)
+        elif value.__name__ == "hints_wrapper":
+            return WrapHigherOrderVariable(value, source, **kwargs)
```

Comment:
We probably want to create new HintsWrapperHigherOrderVariable. See below comments.